From e094dead91389c4a5f602e5bef2747ad5e73e8bd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 8 Apr 2021 16:38:40 +0200
Subject: [PATCH] windowsNavigator: Adjust to gnome-shell change

Parts of WindowPreview were moved to C for performance reasons, which
turned a formerly private JS property into a GObject property.

https://gitlab.gnome.org/GNOME/gnome-shell-extensions/-/issues/302

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell-extensions/-/merge_requests/170>
---
 extensions/windowsNavigator/extension.js | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/extensions/windowsNavigator/extension.js b/extensions/windowsNavigator/extension.js
index 0ea9645..bb72d7b 100644
--- a/extensions/windowsNavigator/extension.js
+++ b/extensions/windowsNavigator/extension.js
@@ -84,17 +84,17 @@ class MyWorkspace extends Workspace.Workspace {
             });
 
             this._text.add_constraint(new Clutter.BindConstraint({
-                source: this._windowContainer,
+                source: this.windowContainer,
                 coordinate: Clutter.BindCoordinate.POSITION,
             }));
             this._text.add_constraint(new Clutter.AlignConstraint({
-                source: this._windowContainer,
+                source: this.windowContainer,
                 align_axis: Clutter.AlignAxis.X_AXIS,
                 pivot_point: new Graphene.Point({ x: 0.5, y: -1 }),
                 factor: this._closeButtonSide === St.Side.LEFT ? 1 : 0,
             }));
             this._text.add_constraint(new Clutter.AlignConstraint({
-                source: this._windowContainer,
+                source: this.windowContainer,
                 align_axis: Clutter.AlignAxis.Y_AXIS,
                 pivot_point: new Graphene.Point({ x: -1, y: 0.5 }),
                 factor: 0,
-- 
2.26.3

